src/components/public-holidays/public-holidays.tpl.html

```diff
@@ -0,0 +1,13 @@
+You have following Publie Holidays :<br/>
```

Comment:
"You have following Publie Holidays :" why not write something like "Public holidays in current year: "